frontend/src/pages/metrics/index.ts

```diff
@@ -1,59 +1,37 @@
 import "@/pages/sentry";
 import MetricsPage from "@/pages/metrics/Metrics.vue";
 
-const el = document.getElementById("app");
-if (el) {
-  // const metricsData = JSON.parse(el.getAttribute("data-all-metrics-data"))
-  const metricsData = JSON.parse(el.getAttribute("data-metrics-data"));
+export interface TimeSeries {
+  name: string;
+  data: number[];
+  type?: string;
+}
 
-  console.log("metricsData");
-  console.log(JSON.stringify(metricsData, null, 4));
+export interface Metric {
+  title: string;
+  y_label: string;
```

Comment:
should we consider using something like https://humps.readthedocs.io/en/latest/ to camelCase our snake cased python data?

Reply:
Should be all good, I'm guessing highcharts is deciding to not use labels due to some series being too small, however I haven't managed to find a way around this. 

Agreed on the casing though, its something I'd like to resolve in the v3 codebase, I like the idea of doing the conversion server-side though